src/plone/passwordless/browser/templates/practicing.pt

```diff
@@ -1,5 +1,9 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" metal:use-macro="context/main_template/macros/master" i18n:domain="plone.passwordless">
 
+    <metal:block fill-slot="javascript_head_slot">
+        <script type="text/javascript" src="//localhost:8080/Plone/++plone++static/script.js"></script>
```

Comment:
We never use full URLs in templates. 

Better:

`<script type="..." tal:attributes="src string:$portal_url/++resource++plone.passwordless/script.js"></script>
`

`$portal_url` will be replaced automatically by the templating engine with the current URL that comes from the browser.